### PR TITLE
Fix incorrect logging for Buffers

### PR DESCRIPTION
**Pull request by tkiapril**
SSIA

### CODE REVIEW

File: emitter.ts [hunk 1]
```diff
@@ -87,7 +87,11 @@ export async function emitter(
       emitDestinations.length > 0
         ? `Loaded emit destinations, ${
           emitDestinations.map((dest) =>
-            `address: ${dest.sourceAddress}  event signature hash: ${dest.abiHash}  topic filters: [1] ${dest.topic1} [2] ${dest.topic2} [3] ${dest.topic3}  destination: ${dest.webhookUrl}`
+            `address: ${toHex(dest.sourceAddress)}  event signature hash: ${
+              toHex(dest.abiHash)
+            }  topic filters: [1] ${toHex(dest.topic1)} [2] ${
+              toHex(dest.topic2)
+            } [3] ${toHex(dest.topic3)}  destination: ${dest.webhookUrl}`
```
Comment (eseiker) [inline emitter.ts:94]: Seems nullish check for topics is needed before invoking toHex

(+ I would prefer the way like L300 below)

File: emitter.ts [hunk 2]
```diff
@@ -288,7 +292,11 @@ export async function emitter(
 
         if (event == null) {
           logger.error(() =>
-            `Event does not exist in DB, blockNumber-txIndex-logIndex: ${x.blockTimestamp}-${x.txIndex}-${x.logIndex}  blockHash: ${x.blockHash}  address: ${x.address}  event signature hash: ${x.sigHash}  topics: ${
+            `Event does not exist in DB, blockNumber-txIndex-logIndex: ${x.blockTimestamp}-${x.txIndex}-${x.logIndex}  blockHash: ${
+              toHex(x.blockHash)
+            }  address: ${toHex(x.address)}  event signature hash: ${
+              toHex(x.sigHash)
+            }  topics: ${
               x.topics.map((topic, i) => `[${i}] ${toHex(topic)}`).join(" ")
```
Comment (eseiker) [inline emitter.ts:300]: Missed from previous PR, topic seems to be nullish checked before calling toHex, also.

(+ [${i + 1}] maybe?)

**Review by eseiker — COMMENTED** [review]
not an issue but just whining.. I don't like the way what deno fmt have done here...
too sad that JS/TS still haven't land string literal dedent feature natively...
hope there were better way to build long string without linebreaks

File: emitter.ts (hunk 1 above)
Comment (tkiapril) [reply emitter.ts]: Amended!

File: emitter.ts (hunk 2 above)
Comment (tkiapril) [reply emitter.ts]: Amended!

**Review by eseiker — APPROVED** [review]

**Pull request merged**